### ISSUE TITLE
Moving photo pin

### ISSUE THREAD

**Issue by chdot** [cyclestreets/ios#69]
When dragging pin to correct location it stops when it reaches the grey dialog box and finger has to be used to move map and then pin moved more. 

Can this be changed to continuous movement?

**Comment by mvl22** [comment]
Is this possible to fix in the new Mapkit implementation?

**Comment by neilkachu** [comment]
What they would like is for the map to move as the pin reaches the edges of the current map boundary so they are doing both actions at once. This is not natively doable, we could probably add a timer with a check for a edge area and then shift the map. There maybe a couple of issues: a) can we get the annotation location whilst dragging (if not we may be able to use the actual touch point) and b) if you move the map under the annotation will mapkit see this as a touch up as the map/annotation sync will be changed. I'll need to investigate but might be doable.

**Comment by mvl22** [comment]
> What they would like is for the map to move as the pin reaches the edges of the current map boundary so they are doing both actions at once.

Fair enough, but at least adding support for basic draggability in the first place would be a clear benefit.

**Comment by neilkachu** [comment]
We already allow dragging, if you mean the location view screen this is for display only as it does not allow resetting.

**Comment by mvl22** [comment]
> We already allow dragging

I mean the stage of the wizard where the location can be changed (i.e. where it gives you the default location from the photo EXIF, and you can move the marker if it's wrong).

That definitely doesn't let me drag.

**Comment by neilkachu** [comment]
Which one, a)

![img_1105](https://cloud.githubusercontent.com/assets/476062/12883485/467e682c-ce50-11e5-9698-719b21354bc8.PNG)

or b)
![img_1104](https://cloud.githubusercontent.com/assets/476062/12883482/4335ee1a-ce50-11e5-949b-ca259d335215.PNG)

b definitely allows dragging, a is for display only.
